team.html

```diff
@@ -150,7 +102,7 @@ <h4 class="card-heading">Sebastian<br>Peter</h4>
             </div>
             <div class="card-content-block"><img src="images/work.svg" loading="lazy" id="w-node-ce389b2d-6a69-c457-e8c3-500d801ac5b5-98e4b81b" alt="">
               <p id="w-node-ce389b2d-6a69-c457-e8c3-500d801ac5b6-98e4b81b" class="md">Research Associate</p><img src="images/graduation-hat.svg" loading="lazy" id="w-node-ce389b2d-6a69-c457-e8c3-500d801ac5b8-98e4b81b" alt="">
-              <p id="w-node-ce389b2d-6a69-c457-e8c3-500d801ac5b9-98e4b81b" class="md">M. Sc. Electrical Engineering</p><img src="images/research.svg" loading="lazy" alt="">
+              <p id="w-node-ce389b2d-6a69-c457-e8c3-500d801ac5b9-98e4b81b" class="md">M. Sc. Computer Science</p><img src="images/research.svg" loading="lazy" alt="">
```

Comment:
Super pedantic point and I hate to bring this up, but the way that degrees like `M.Sc.` are written differently here (space in between). It would calm my pedantic soul if we would write them all in the same way. I think the version without a space in between is more common.